metrics/resizer.go

```diff
@@ -0,0 +1,71 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	runtimemetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// Metrics subsystem and all of the keys used by the resizer.
+const (
+	ResizerSuccessLoopTotalKey = "success_loop_total"
+	ResizerFailedLoopTotalKey  = "failed_loop_total"
+	ResizerLoopSecondsTotalKey = "loop_seconds_total"
+)
+
+func init() {
+	registerResizerMetrics()
+}
+
+type resizerSuccessLoopTotalAdapter struct {
+	metric *prometheus.CounterVec
+}
+
+func (a *resizerSuccessLoopTotalAdapter) Increment(ns, name string) {
+	a.metric.WithLabelValues(ns, name).Inc()
+}
+
+type resizerFailedLoopTotalAdapter struct {
+	metric *prometheus.CounterVec
+}
+
+func (a *resizerFailedLoopTotalAdapter) Increment(ns, name string) {
+	a.metric.WithLabelValues(ns, name).Inc()
+}
+
+type loopSecondsTotalAdapter struct {
+	metric prometheus.Counter
+}
+
+func (a *loopSecondsTotalAdapter) Add(value float64) {
+	a.metric.Add(value)
+}
+
+var (
+	resizerSuccessLoopTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: MetricsNamespace,
+		Name:      ResizerSuccessLoopTotalKey,
+		Help:      "counter that indicates how many volume expansion processing loops succeed.",
+	}, []string{"namespace", "name"})
```

Comment:
`namespace` and `name` labels are highly likely to conflict with other relabeled labels since it is too general. Would you add a prefix like `pvc_`?